opencti-platform/opencti-graphql/src/manager/ingestionManager.ts

```diff
@@ -271,6 +275,80 @@ const taxiiExecutor = async (context: AuthContext) => {
 };
 // endregion
 
+// region Csv ingestion
+interface CsvResponseData {
+  data: string,
+  addedLast: string | undefined | null
+}
+const csvHttpGet = async (ingestion: BasicStoreEntityIngestionCsv): Promise<CsvResponseData> => {
+  const headers = new AxiosHeaders();
+  headers.Accept = 'application/csv';
+  if (ingestion.authentication_type === 'basic') {
+    const auth = Buffer.from(ingestion.authentication_value, 'utf-8').toString('base64');
+    headers.Authorization = `Basic ${auth}`;
+  }
+  if (ingestion.authentication_type === 'bearer') {
+    headers.Authorization = `Bearer ${ingestion.authentication_value}`;
+  }
+  let certificates;
+  if (ingestion.authentication_type === 'certificate') {
+    certificates = { cert: ingestion.authentication_value.split(':')[0], key: ingestion.authentication_value.split(':')[1], ca: ingestion.authentication_value.split(':')[0] };
+  }
+  const httpClientOptions: GetHttpClient = { headers, rejectUnauthorized: false, responseType: 'json', certificates };
+  const httpClient = getHttpClient(httpClientOptions);
+  const { data, headers: resultHeaders } = await httpClient.get(ingestion.uri);
+  return { data, addedLast: resultHeaders['x-csv-date-added-last'] };
+};
+const csvDataToObjects = async (data: string, ingestion: BasicStoreEntityIngestionCsv, csvMapper: BasicStoreEntityCsvMapper, context: AuthContext) => { // push bundle to queues
+  const entitiesData = data.split('\n');
+  logApp.info(`[OPENCTI-MODULE] CSV ingestion execution for ${entitiesData.length} items`);
```

Comment:
I suggest you lower the log level to debug, this happens a lot during my tests (and I only had one ingester set !)